src/js/components/modals/CliInstallModal.js

```diff
@@ -67,7 +67,12 @@ class CliInstallModal extends React.Component {
     const clusterUrl = `${protocol}://${hostname}${port}`;
     const { selectedOS } = this.state;
 
-    const downloadUrl = `https://downloads.dcos.io/binaries/cli/${osTypes[selectedOS]}/x86-64/dcos-1.13/dcos`;
+    // Starting from DC/OS >=2.0, users can stick to the "latest" CLI.
+    // However on DC/OS 1.x we must use the correct CLI for the given version of DC/OS.
+    let dcosVersion = MetadataStore.parsedVersion
+    cliVersion = dcosVersion.startsWith('1.') ? `dcos-${dcosVersion}` : "latest";
```

Comment:
I don't know whether the UI provides a polyfill for `startsWith`, otherwise this will not work in IE: https://caniuse.com/#search=startswith Oldie but goldie.

As a workaround you can use `substring`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substring

Reply:
Updated